region/brinstar/pink.json

```diff
@@ -2361,6 +2371,10 @@
                   "name": "Waterway Suitless Spark",
                   "notable": true,
                   "requires": [
+                    {"or": [
```

Comment:
It's possible to get up there with a SpringBall Jump (having HiJump) by jumping a tiny bit later than you press pause.

If getting on the ledge is no longer free, we also have to consider a `canComeInCharged` from node 1. Needs roughly 60 frames remaining on it, and it's a ~150-frame shinespark.